SOTS_2019_WDR.tex

```diff
@@ -0,0 +1,502 @@
+        \item Parallel Fill Valve
+        \item Series Fill Valve
+        \item Line Vent Valve
+        \item Parallel Vent Valve
+    \end{checklist}
+    \item \ops{}: Confirm that the Tank Vent Valve is initially open.
+    \item \ops{}: Confirm that the Pressure Relief Valve is initially closed.
+    \item \ops{}: Confirm that the Injector Valve is initially closed.
+    \item \primary: Slowly open the Cylinder Valve through $\frac{3}{4}$ of a turn.
+    \begin{checklist}[label=$\bullet$]
+        \item If leaks are observed:
+        \begin{checklist}
+            \item \ops{}: Proceed to procedure A1.
+        \end{checklist}
+    \end{checklist}
+    \item \primary{}: Communicate the supply line pressure as visible on the Pressure Gauge.
+    \begin{checklist}[label=$\bullet$]
+        \item If the supply line pressure is below 800 psi:
+        \begin{checklist}
+            \item \ops{}: Proceed to procedure \textbf{A2}.
+        \end{checklist}
+        \item If the supply line pressure exceeds 1000 psi:
+        \begin{checklist}
+            \item \ops{}: Proceed to procedure \textbf{A3}.
+        \end{checklist}
+    \end{checklist}
+    \item \control: Confirm that the supply line pressure as read by \primary{} agrees with the supply line pressure [P1] measured by the DAQ system.
+    \item \ops{}: Record the resting rocket dry mass and supply pressure:
+
+    \begin{tabu} to \textwidth { |[1pt]X[c]|[1pt]X[c]|[1pt] }
+    \tabucline[1pt]{}
+    [M] Dry Mass (lbs) & [P1] Supply Pressure (psi) \\
+    \tabucline[1pt]{}
+    \vspace{1in} & \\
+    \tabucline[1pt]{}
+    \end{tabu}
+
+    \item \primary: Open the Series Fill Valve.
+    \item \primary{} and \secondary{}: Retreat 100 ft from the fill system.
+    \item \ops{}: Give the system control key to \control{}.
+    \item \control{}: Confirm the following valves are closed:
+    \begin{checklist}
+        \item Remote Fill Valve
+        \item Line Vent Valve
+        \item Tank Vent Valve
+    \end{checklist}
+    \item \control{}: Engage the key switch and enable actuators.
+    \item \control{}: Open and close the Tank Vent Valve, ensuring that the limit switch reading updates accordingly.
+    \item \control{}: Open the Remote Fill Valve.
+    \item \control{}: Confirm the following pressures are increasing:
+    \begin{checklist}
+        \item {[P2]} Fill line pressure
+        \item {[P3]} Oxidizer tank pressure
+    \end{checklist}
+    \item \control{}: Close the Remote Fill Valve.
+    \item \control{}: Confirm the following pressures are stable:
+    \begin{checklist}
+        \item {[P2]} Fill line pressure
+        \item {[P3]} Oxidizer tank pressure
+    \end{checklist}
+    \begin{checklist}[label=$\bullet$]
+        \item If the pressures are decreasing:
+        \begin{checklist}
+            \item \ops{}: Proceed to procedure \textbf{A4}.
+        \end{checklist}
+    \end{checklist}
+    \item \control{}: Open the Line Vent Valve.
```

Comment:
We need to actuate disconnect after we've done this step, which means we need to:

- Confirm the fill line is atmospheric
- Actuate remote disconnect
- Confirm that the limit switch says we've disconnected